2215.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @param {number[]} nums1
+ * @param {number[]} nums2
+ * @return {number[][]}
+ */
+var findDifference = function (nums1, nums2) {
+  const ans = [];
+  const ans0 = [];
+  const ans1 = [];
```

Comment:
It's good to give variables/constants self-descriptive names as much as you can for readability.
These look fine but you should give self-descriptive names if you can think of them.